examples/tree_2d_dgsem/elixir_advection_diffusion.jl

```diff
@@ -47,6 +47,7 @@ boundary_conditions_parabolic = boundary_condition_periodic
 semi = SemidiscretizationHyperbolicParabolic(mesh,
                                              (equations, equations_parabolic),
                                              initial_condition, solver;
+                                             solver_parabolic = ViscousFormulationBassiRebay1(),
                                              boundary_conditions = (boundary_conditions,
                                                                     boundary_conditions_parabolic))
```

Comment:
Just to be sure: This is to be concrete about the setup but not required since the default method will still give `solver_parabolic = ViscousFormulationBassiRebay1()`, won't it?

Reply:
As I understood, it's mostly to be able to overwrite `solver_parabolic` in `@test_trixi_include` in the tests below.

Reply:
Yes, exactly @JoshuaLampert